packages/framework/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,85 @@
+{
+	"name": "@fluidframework/app-insights-logger",
```

Comment:
Nit: It might be a good idea to start with this package being set to `private`. We can do a separate PR to make it public once we are confident that everything is working as we want. As soon as we mark it as public, we are making compatibility guarantees.

Reply:
Okay, I got a policy violation for it, apparently they have to be public in the framework package. There might be a way to add an exclusion

Reply:
An alternative would be to put it in `@fluid-experimental` first. I remember seeing some discussion about that being an ok starting path to publish new packages, and move them to `@fluidframework` once it's proven itself.